stock_mts_mto_mrp_rule/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Stock MTS+MTO MRP Rule",
+    "summary": "Stock MTS+MTO MRP Rule",
+    "version": "14.0.1.0.0",
+    "development_status": "Mature",
```

Comment:
IMHO, it's quite soon to set it like that.
Remove the entry as default one is Beta.

Reply:
Resolved